Assets/MixedRealityToolkit.Extensions/SpectatorView/Scripts/Compositor/ICalibrationData.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+namespace Microsoft.MixedReality.Toolkit.Extensions.Experimental.SpectatorView.Compositor
+{
+    public interface ICalibrationData
```

Comment:
>ICalibrationData [](start = 21, length = 16)

Does something currently implement this interface that may be missing from this review?

Reply:
I didn't hook up calibration ingestion yet (since that requires a camera providing poses to actually test) but I didn't want to remove ICalibrationData or some of that functionality in CompositionManager since it'll be coming in a PR very soon.